doc/jar_converter/readme.md

```diff
@@ -21,6 +22,12 @@ Nachdem die Dateien und Verzeichnisse kopiert wurden, erstellt das Skript eine J
    ./jarConverter.sh datei1.dng verzeichnis1 datei2.dng
    ```
 
+oder unter Windows
+
+   ```batch
+   ./jarConverter.bat datei1.dng verzeichnis1 datei2.dng
```

Comment:
Klickt man unter Windows nicht auf irgendwas drauf? Würde man die Bat wirklich von Hand in der Konsole starten?

Reply:
ich hab keine Ahnung wie man dann Command-Line Arguments übergibt. 
Ich glaube dafür brauchst du schon ein Terminal.

Reply:
ja gut. war ja nur ne frage. ich habe schon ewig kein windows mehr angefasst :)

Reply:
musst mal wieder windows putzen

Reply:
indeed.

Reply:
In der klassischen CMD unter Windows wird der Aufruf `./jarConverter.bat` nicht funktionieren. Stattdessen muss man einfach direkt `jarConverter.bat` aufrufen.
In der PowerShell (und im Terminal von IntelliJ) ist es genau andersrum. Das wäre vielleicht noch sinnvoll zu dokumentieren.